src/l500/l500-color.cpp

```diff
@@ -485,6 +499,62 @@ namespace librealsense
     }
 
 
+    void l500_color_sensor::set_default_controls_for_calibration_if_needed()
+    {
+        for (auto && calib_control : _calib_controls)
+        {
+            auto && control = get_option(calib_control.option);
+
+            auto curr_val = control.query();
+            if (curr_val != calib_control.default_value)
+            {
+                LOG_INFO( "Calibration changed option: " << rs2_option_to_string( calib_control.option )
```

Comment:
Make this AC_LOG so we see it there... pretty important.

Reply:
And these should not be visible in the Viewer... please make debug.

Reply:
> AC_LOG - Right I will change
> DEBUG - Why not? It's an info to the user that we change controls, He will not think the application is acting weird, 
                   It's the same like we write that we open the color sensor, we are automatically changing things

Reply:
Only really important messages should be shown to the user -- start/stop AC, and errors

Reply:
Done